tables/Cuneiform-transliteration.utd

```diff
@@ -0,0 +1,186 @@
+
+# liblouis: Cuneiform-Transliteration
+#
+# ------------
+#-index-name: Cuneiform-Transliteration uncontracted
+#-display-name: Cuneiform-Transliteration uncontracted
+#
+#+language:Cuneiform-transliteration
+#+type:literary
+#+contraction:no
+#+system:Cuneiform-transliteration
+
+#-license: lgpl-2.1
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+
+#This transliteration table provides support for the transliteration of all Cuneiform languages. For more information on the Transliteration table, please go to:
+#http://oracc.ub.uni-muenchen.de/doc/help/languages/ugaritic/index.html
+#The standard for Transliteration has been set by the academic community represented by ORACC. The braille code for transliteration follows the standard set by ORACC. The braille code for transliteration is represented in braille as the name for the sign in the transliteration table. Thus, if the sign is a "a," then the braille code for this sign would be a.
```

Comment:
Same comment as above about the wording. I find "this transliteration table" and "braille code for transliteration" a bit confusing.